src/rendermime/index.ts

```diff
@@ -6,6 +6,10 @@ import {
 } from 'phosphor/lib/core/token';
 
 import {
+  JSONObject
+} from 'phosphor/lib/algorithm/json';
```

Comment:
We ascii order imports based on module name. This should go before `phosphor/lib/core`.